readme.md

```diff
@@ -60,6 +60,12 @@ Type: `number | Iterable<number>`
 
 A preferred port or an iterable of preferred ports to use.
 
+##### exclude
+
+Type: `Iterable<number>`
+
+An iterable of ports which we do not want to use
```

Comment:
The docs here should be in sync with the TS docs.

Reply:
Can you help me understand where are the TS Docs? are you referring to the .d.ts file?

Reply:
Yes he is